### PR TITLE
update

### PR DESCRIPTION
**Pull request by hagan199**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- Stylesheet -->
```
Comment (GedeonTS) [inline index.html:6]: - [OPTIONAL] it's always better to avoid having commented lines of code to make your code cleaner :+1:

Comment (GedeonTS) [inline index.html:6]: - [OPTIONAL] it's always better to avoid having commented lines of code to make your code cleaner :+1:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @hagan199 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 

- Good use of GitHub flow :+1: 

## Required Changes ♻️
- [ ] As per the project requirements, kindly make sure your project has passing linter checks
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (hagan199) [reply index.html]: Remove commented cod

Comment (hagan199) [reply index.html]: Remove commented code

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# .gitignore
```
[the remark this answers is not in the document]
Comment (hagan199) [reply .gitignore:1]: Added to ignore

File: README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+
+<div align="center">
+
+  <img src="smalllogo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
+
```
[the remark this answers is not in the document]
Comment (hagan199) [reply README.md:10]: Remove

**Pull request closed without merging**